src/sentry/static/sentry/app/components/button.tsx

```diff
@@ -99,6 +99,8 @@ class Button extends React.Component<ButtonProps, {}> {
 
     // Don't allow clicks when disabled or busy
     if (disabled || busy) {
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
Would love to be able to capture this behavior in a test, but I don't think we can with enzyme since we don't have real events (and thus can't test behavior with a `<form>` and `<Button>`. I don't think  using `simulate()` with a mocked event is really useful.